### PR TITLE
Fix detection of asynchronous start of dev appserver for compat apps

### LINKED ISSUE #172: Dev App Server start is not properly detected for compat apps
The message in the log that indicates that App Engine app with `vm=true` has started on Dev App Server is `INFO:oejs.Server:main: Started`.

### PR DESCRIPTION
**Pull request by meltsufin**
Fixes #172 

@loosebazooka

### CODE REVIEW

File: src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java [hunk 1]
```diff
@@ -257,7 +258,7 @@ public void validate() throws AppEngineException {
     if (!Files.isRegularFile(JAR_LOCATIONS.get(JAVA_TOOLS_JAR))) {
       throw new AppEngineException(
           "Validation Error: Java Tools jar location '"
-                  + JAR_LOCATIONS.get(JAVA_TOOLS_JAR) + "' is not a file.");
```
Comment (loosebazooka) [inline src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java]: Actually not sure what the correct spacing is here

Comment (meltsufin) [inline src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java]: Me neither, I just did autoformat from intellij.

Comment (meltsufin) [inline src/main/java/com/google/cloud/tools/appengine/cloudsdk/CloudSdk.java]: OK, so I looked it up in the [Google Code Style Guide](https://google.github.io/styleguide/javaguide.html#s4.5.2-line-wrapping-indent), and the correct continuation indent is 4 or more spaces. I think everywhere else in the code, we're using 4 spaces. So, let's stick with that.

**Comment by loosebazooka** [comment]
:ship:

**Pull request merged**